projects/packages/forms/src/dashboard/integrations/google-sheets-card.tsx

```diff
@@ -17,6 +14,8 @@ const GoogleSheetsDashboardCard = ( {
 }: IntegrationCardProps ) => {
 	const isConnected = !! data?.isConnected;
 	const settingsUrl = data?.settingsUrl;
+	const FORM_RESPONSES_URL =
```

Comment:
same for this. Could we not just use `config( 'dashboardURL' )` ?

Reply:
Same as above, the PR is just updating the menu slug references, this time I thought it was good to use the config export so we bring the value from the ported dataset we print on backend, but the fallback is there just in case and the real update is just the menu slug.